pom.xml

```diff
@@ -41,8 +41,9 @@
     <!-- DataSource (HikariCP) -->
     <dependency>
       <groupId>com.zaxxer</groupId>
-      <artifactId>HikariCP</artifactId>
-      <version>2.4.3</version>
+      <artifactId>HikariCP-java7</artifactId>
+      <version>2.4.13</version>
+      <scope>compile</scope>
```

Comment:
some downstream projects will be java8, will they have a problem with this dependency if they already include hikari (are the classes the same in hikari java8 and java7)?
Perhaps this should be provided scope so downstream project can pick correct dep. for their project.

Reply:
I am ok going for this and exclude if needed as an alternative as well btw.